qiita_pet/test/test_admin_processing_job_handlers.py

```diff
@@ -45,5 +45,19 @@ def test_get_missing_argument(self):
                       response.body.decode('ascii'))
 
 
+class TestSampleValidation(BaseAdminTests):
+    def test_get(self):
+        response = self.get('/admin/sample_validation/')
+        self.assertEqual(response.code, 200)
+
+    def test_post(self):
+        post_args = {
+            'qid': 1,
+            'snames': 'SKB1.640202 SKB2.640194 BLANK.1A BLANK.1B'
+        }
+        response = self.post('/admin/sample_validation/', post_args)
+        self.assertEqual(response.code, 200)
```

Comment:
Some coders will return 200 as long as the command executed without failure on the server-side. It's good to check the response.text and confirm it does not contain error messages and contains at least some string of text you're expecting.

Reply:
Modified unit test to check that the response body contains the inputted sample names